build-system/npm-publish/test/write-package-files.test.js

```diff
@@ -0,0 +1,40 @@
+const test = require('ava');
+const {existsSync, mkdirSync, rmSync, writeFileSync} = require('fs-extra');
+const {exportedForTesting} = require('../write-package-files');
+
+const {getDir, writePackageFiles} = exportedForTesting;
+
+let createdFiles = [];
+
+test.before(() => {
+  createdFiles = [];
+});
+test.after(() => {
+  createdFiles.forEach((file) => {
+    rmSync(file, {force: true});
+  });
+});
+
+test('write-package-files', async (t) => {
```

Comment:
similar to above - can we also test writing package files for an "extension" as well?